pom.xml

```diff
@@ -47,19 +49,15 @@
         <tag>HEAD</tag>
         <url>https://github.com/apifocal/apifocal-parent</url>
     </scm>
-    <ciManagement>
-        <system>Jenkins</system>
-        <url>https://jenkins.apifocal.org/job/Dev/job/apifocal-parent/</url>
-    </ciManagement>
 
     <distributionManagement>
         <repository>
-            <id>apifocal-releases</id>
```

Comment:
I think we should leave apifocal repository definitions towards archiva.apifocal.org and add ossrh. Would that be a problem?

Reply:
The only problem is that maven doesn't allow for multiple repositories in distributionManagement. I agree with the intention, but doesn't work.

Reply:
This branches our work into two sections: one that is using OSSRH and the other using archiva.apifocal.org. I guess OSSRH is supposed to be open-source, much like the name suggests.

Maybe have two parents? `apifocal-oss` for open-source and `apifocal-css` for closed-source software?

So far all projects rely on `apifocal-parent` and all docker images are built using archiva.apifocal.org. How do you envision to propagate this change? Reparent silkmq to `apifocal-css`, which is parented with `apifocal-parent` (the OSS one) ? Or have a common `apifocal-parent` with two child poms?

Reply:
Looks like there's a better way, I'll get it done.